install_scripts/getSimId.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var exec = require('child_process').exec;
+var path = require('path');
+
+var SP = require('serialport');
+var SerialPort = SP.SerialPort;
+
+var privatePath = path.join(__dirname, '..', 'PRIVATE.json');
+
+exec('cat ' + privatePath + ' | grep -e \'"sim": "undefined"\'', function (err, stdout) {
+	if (err || stdout.length === 0)
+		throw new Error('Cannot find an undefined field in the PRIVATE.json');
```

Comment:
`null` is a valid JSON valid IIRC (actual `null`, not just the string `"null"`). Probably a better choice than `"undefined"`